numpy/tests/test_numpy_config.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
I realised after pushing, this test should be restricted to meson builds for now. Will look into it.

Reply:
Edited old commit to 36d865d97e201b54b14071dabcd8553ce72d5081